install_config/monitoring/configuring-openshift-cluster-monitoring.adoc

```diff
@@ -0,0 +1,95 @@
+:data-uri:
+:icons:
+:experimental:
+:prewrap!:
+
+The OpenShift Ansible `openshift_cluster_monitoring_operator` role configures and deploys the Cluster Monitoring Operator using the variables from the xref:../install/configuring_inventory_file.adoc#configuring-ansible[inventory file].
+
+.Ansible variables
+[options="header"]
+|===
+
+|Variable |Description
+
+|`openshift_cluster_monitoring_operator_install`
+| Deploy the Cluster Monitoring Operator if `true`. Otherwise, undeploy. This variable is set to `true` by default.
+
+|`openshift_cluster_monitoring_operator_prometheus_storage_capacity`
+| The persistent volume claim size for each of the Prometheus instances. This variable applies only if `openshift_cluster_monitoring_operator_prometheus_storage_enabled` is set to `true`. Defaults to `50Gi`.
+
+|`openshift_cluster_monitoring_operator_alertmanager_storage_capacity`
+| The persistent volume claim size for each of the Alertmanager instances. This variable applies only if `openshift_cluster_monitoring_operator_alertmanager_storage_enabled` is set to `true`. Defaults to `2Gi`.
+
+|`openshift_cluster_monitoring_operator_node_selector`
+| Set to the desired, existing xref:../admin_guide/scheduling/node_selector.adoc#admin-guide-sched-selector[node selector] to ensure that pods are placed onto nodes with specific labels. Defaults to `node-role.kubernetes.io/infra=true`.
+
+|`openshift_cluster_monitoring_operator_alertmanager_config`
+| Configures Alertmanager. See xref:configuring-alertmanager.adoc#configuring-alertmanager[Configuring Alertmanager]
+
+|`openshift_cluster_monitoring_operator_prometheus_storage_enabled`
+| Enable persistent storage of Prometheus' time-series data. This variable is set to `false` by default.
+
+|`openshift_cluster_monitoring_operator_alertmanager_storage_enabled`
+| Enable persistent storage of Alertmanager notifications and silences. This variable is set to `false` by default.
+
+|===
+
+[[monitoring-prerequisites]]
+== Monitoring prerequisites
+
+The monitoring stack imposes additional resource requirements. See xref:../scaling_performance/scaling_cluster_monitoring.adoc#cluster-monitoring-recommendations-for-OCP[computing resources recommendations] for details.
+
+== Installing monitoring stack
+
+The Monitoring stack is installed with OpenShift by default. You can prevent it from being installed. To do that, set this variable to `false` in the Ansible inventory file:
+
+`openshift_cluster_monitoring_operator_install`
+
+== Persistent storage
+
+Running cluster monitoring with persistent storage means that your metrics are stored to a xref:../architecture/additional_concepts/storage.adoc#persistent-volumes[persistent volume] and can survive a pod being restarted or recreated. This is ideal if you require your metrics or alerting data to be guarded from data loss. For production environments it is highly recommended to configure persistent storage.
+
+=== Enabling persistent storage
```

Comment:
I think you need a set of substeps to modify the inventory file. Something like:

. Modify the inventory file to provide cluster monitoring storage:
.. All
.. Your
.. Storage
.. Steps